chore: a note about the archive label (#216)
mdn/mdn-community: README.md

| 🛠️  [Platform](https://github.com/mdn/mdn-community/discussions/categories/platform) | Discussions related to the [core](https://github.com/mdn/yari) MDN Web Docs platform. Your suggestions to improve the architecture and existing features, such as navigation and search, are welcome. However, if you believe you found a bug related to the platform, please [report it](https://github.com/mdn/yari/issues/choose) on the Yari repository. This is also the place for discussions related to existing tooling, such as [BoB](https://github.com/mdn/bob), [markdown](https://github.com/mdn/markdown/), reusable [workflows](https://github.com/mdn/workflows), etc.  _NOTE:_ This category is not for MDN Plus-related feature discussions. There is a separate discussion category for [MDN Plus]((https://github.com/mdn/mdn-community/discussions/categories/mdn-plus)). |
| 🤖  [Polls](https://github.com/mdn/mdn-community/discussions/categories/polls) | This category is meant for use by the MDN Web Docs staff. We will use this category to run polls around topics where we need your input. So, keep your eyes peeled 👀 |

> > **A note on discussions labelled as archived**: We label some discussions with an `archive` label. This is because sometimes we find two or more discussions open around the same topic. Currently, you cannot merge two discussions into one, so our current workaround is to link to the least active discussion(s) from the most active. We will lock the older discussions and add the `archived` label. This allows us to keep a record of the older discussions but focus on, the more active discussion.

Reviewer: ```suggestion
> **A note on discussions labelled as archived**: We label some discussions with an `archive` label because we sometimes find two or more discussions around the same topic. Currently, you cannot merge two discussions into one; our workaround is to add a link in the most active discussion that references the least active discussion(s). We will lock older discussions and add the `archived` label, which allows us to keep the older discussion history but focus on active discussions.
```
